python/paddle/fluid/optimizer.py

```diff
@@ -268,6 +317,9 @@ def minimize(self,
 
         optimize_ops = self._create_optimization_pass(params_grads, loss,
                                                       startup_program)
+        if table_optimize_op is not None:
+            optimize_ops.append(table_optimize_op)
```

Comment:
output warning if this change is taking effect?

Reply:
Ok, will add in following PR.